### PR TITLE
task 5 done and updated JSON file for endpoint /api

### PR DESCRIPTION
**Pull request by yaahcd**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -22,4 +22,19 @@ afterAll(() => {
...
+        expect(article).toHaveProperty('article_id', expect.any(Number));
+        expect(article).toHaveProperty('title', expect.any(String));
+        expect(article).toHaveProperty('topic', expect.any(String));
+        expect(article).toHaveProperty('author', expect.any(String));
+        expect(article).toHaveProperty('body', expect.any(String));
+        expect(article).toHaveProperty('created_at', expect.any(String));
+        expect(article).toHaveProperty('article_img_url', expect.any(String));
+      })
+    })
+  })
```
Comment (Slyose) [inline __tests__/app.test.js]: Nicely tested so far! One more thing you need is a test to ensure that the articles are be sorted by date in descending order. We highly recommend using the [Jest sorted](https://www.npmjs.com/package/jest-sorted) npm package for this as it has sorting functionality built in.

Also, the task states "There should not be a body property present on any of the article objects." which you need to account for also!

**Review by Slyose — COMMENTED** [review]
Love what you've done so far! However it needs a few more considerations for you to implement, then please put in another nchelp pr so we can check them.

**Review by Slyose — APPROVED** [review]
Nailed it!

EDIT: noticed a missing test

File: __tests__/app.test.js [hunk 2]
```diff
@@ -31,4 +31,20 @@ afterAll(() => {
...
+        expect(body).toBeSortedBy('created_at',{descending: true})
+        body.forEach((article)=>{
+        expect(article).toHaveProperty('article_id', expect.any(Number));
+        expect(article).toHaveProperty('title', expect.any(String));
+        expect(article).toHaveProperty('topic', expect.any(String));
+        expect(article).toHaveProperty('author', expect.any(String));
+        expect(article).toHaveProperty('created_at', expect.any(String));
+        expect(article).toHaveProperty('article_img_url', expect.any(String));
+        expect(article).toHaveProperty('comment_count', expect.any(String));
+      });
```
Comment (Slyose) [inline __tests__/app.test.js]: You forgot to test that it has no body property. Add that test in, and this ticket is perfect

**Review by Slyose — COMMENTED** [review]
One missing test, almost there

**Review by RichardNeat — APPROVED** [review]
Great stuff Yara - go ahead and merge :)

**Pull request merged**